src/main/java/database/Task.java

```diff
@@ -1,5 +1,76 @@
 package database;
 
+import java.time.LocalDate;
+import java.util.ArrayList;
+
 public class Task {
-    
+    private int id;
+    private String name;
+    private String description;
+    private String inputDate;
+    LocalDate date = LocalDate.parse(inputDate);
+    private enum Status {
+        ToDo,
+        InProgress,
+        Complate
+    }
+    ArrayList<String> comment = new ArrayList<String>();
+    Task subTask = new Task();
```

Comment:
do not use instance of Task class, just use identifier for that, I mean identifier which will reflect real parent or substask. hmm it should be parent task